linalg/hypre.cpp

```diff
@@ -4167,6 +4867,9 @@ void HypreAMS::Init(ParFiniteElementSpace *edge_fespace)
    HYPRE_AMSSetBetaAMGOptions(ams, amg_coarsen_type, amg_agg_levels, amg_rlx_type,
                               theta, amg_interp_type, amg_Pmax);
 
+   HYPRE_AMSSetAlphaAMGCoarseRelaxType(ams, amg_rlx_type);
+   HYPRE_AMSSetBetaAMGCoarseRelaxType(ams, amg_rlx_type);
```

Comment:
This may change the behavior relative to `master` if these two calls actually change anything when `amg_rlx_type` is 8 (CPU hypre build). @tzanio, @liruipeng, is this the case?

Reply:
As far as I can tell in hypre, the default coarse_relax_type = 8, so it may not change the behavior of your CPU build.

Reply:
https://github.com/hypre-space/hypre/blob/e7e2ab36a601e22353931f8d1798e13938330154/src/parcsr_ls/ams.c#L829

Reply:
That's correct, the default in hypre should be `8`.

Reply:
Okay, we should be good then,